packages/http/src/mocker/callback/callbacks.ts

```diff
@@ -126,3 +157,12 @@ function assembleHeaders(
     )
   );
 }
+
+function reduceRange(rangeOrNumber: RangeOrNumber): number {
+  return pipe(
+    rangeOrNumber,
+    Option.fromPredicate(Array.isArray),
+    Option.map(range => range[0] + Math.round(Math.random() * (range[1] - range[0]))),
+    Option.getOrElse(() => rangeOrNumber as number),
```

Comment:
I guess TypeScript is not able to understand that this is a number because `fromPredicate` failed, right?

Reply:
Yes, despite the fact `Array.isArray` is a nice type guard it is not smart enough to imply that here we are definitely not dealing with `[number,number]`.